scm/src/run_scm.py

```diff
@@ -9,44 +9,73 @@
 import subprocess
 import sys
 import time
-from default_namelists import default_physics_namelists
-from default_tracers import default_tracers
+from suite_info import suite, suite_list
 from netCDF4 import Dataset
 
 ###############################################################################
 # Global settings                                                             #
 ###############################################################################
 
-# Name of the Fortran executable to run, including path (relative to run dir)
-EXECUTABLE = './scm'
+SCM_ROOT = ''
+SCM_RUN = ''
+SCM_BIN = ''
+EXECUTABLE = ''
 
-# Path to the directory containing experiment namelists (relative to run dir)
-CASE_NAMELIST_DIR = '../etc/case_config'
+# Name of the Fortran executable to run, including path
+EXECUTABLE_NAME = 'scm'
 
-# Path to the directory containing tracer configurations (relative to run dir)
-TRACERS_DIR = '../etc/tracer_config'
+# Path to default run directory (relative to scm_root)
+DEFAULT_RUN_DIR = 'scm/run'
```

Comment:
All of the global variables listed at the top basically are "default" whether their names say so or not. I did think about which ones to expose to the command line and there is somewhat of a balance there between easier configurability and complexity for users' use cases. The COPY_EXECUTABLE one in particular was more of something that I was using to test that I left in because it might be useful in the future, not because I expect other users to use it.